core/src/main/java/de/bwaldvogel/mongo/MongoServer.java

```diff
@@ -48,7 +49,15 @@
     private SslContext sslContext;
 
     public MongoServer(MongoBackend backend) {
-        this.backend = backend;
+        this(backend, false);
+    }
+
+    public MongoServer(MongoBackend backend, boolean oplogEnabled) {
```

Comment:
I'm not sure the `oplog` flag should be set on constructor (it is too intrusive). 

If user wants to enable ops log he can do:
```java
MongoServer  server new OpslogDecorator(new MongoServer());
```